### DISCUSSION TITLE
Is My Crazyflie Drone Broken?

### DISCUSSION THREAD

**Discussion by akshaysunkara** [bitcraze/discussions#1462] [Q&A]
Hey guys!

For some reason whenever I turn on my crazyflie drone, the M2 light stays permanently blue. I'm able to scan the drone but not able to connect.

Also not able to connect via USB. I really don't know what went wrong PLEASE can someone help because I paid a lot for this drone and dont want it to be broken.

**Comment by akshaysunkara** [comment]
Symptoms:
-When I power up, led M2 lights up solid blue. No other LEDs come on.
-When I power up and hold the button to go into flashing mode, led M2 starts to blink, but led M3 does not.
-The NRF appears to be working as I can detect the flie via bluetooth and with the crazyradio, but connecting never succeeds.
-"make cload" doesn't work after booting to flashing mode
-The crazyflie doesn't enumerate on USB anymore
-The STM32 chip feels a little bit warm to the touch after 15-30 seconds of power

These are the same symptoms that I'm facing as someone else - https://forum.bitcraze.io/viewtopic.php?t=1993

**Reply by akshaysunkara** [reply]
Also, if there is no way to fix this issue, is there any way to get a free replacement from bitcraze or is that not possible?

**Reply by ToveRumar** [reply]
Hi!
Thanks for your question!

I need a bit more information on what has happened. 

It sounds like connecting was working before but now it is not anymore. Is this correct?

When you go into bootloader mode, is this when you can not connect to it? Is it Cold boot(Recovery) you have tried? 
If you did not try cold boot then I suggest doing that. 

Is the drone new? Has it behaved like this since you got it?

It sounds to me that the FW on it might be faulty. 
But of course there is always the potential of it being a HW problem. If we  can not find a way to fix it  we might be able to supply a replacement, depending on the circumstances.

**Reply by akshaysunkara** [reply]
The drone is not new, is has no behaved like this since I've gotten it.

It was connecting and working perfectly before but suddenly stopped working.

I can't connect to it regularly, through this menu, although it scans the drone:

![image](https://github.com/bitcraze/discussions/assets/171630750/0d4d151a-1484-42a4-8827-b11055bb4edc)

I am able to connect to the cold boot recovery menu but whenever I reset it to firmware, and I switch it on, I get the blue LED that I got before that was permament and two other permanent red LEDs.

Would this be an FW or HW problem? 

Also, could you let me know how I would be able to fix it since I work at a lab that does have a soldering station and other stuff

**Reply by akshaysunkara** [reply]
For some reason, I just turned on the drone and it starts up by running all the motors with the propellors in sequence. The M2 light is permanently on and the M1 light is blinking red, stops and then blinks red again in a pattern. 

What does this mean?

**Reply by akshaysunkara** [reply]
The STM32 is still burning my finger every time I touch it, its very very hot

**Reply by evoggy** [reply]
OK, sounds like it might be a hardware issue. Please let us know if the replacement works better.
